modules/oadp-backing-and-restoring-from-cluster-to-cluster.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: CONCEPT
+[id="oadp-backing-and-restoring-from-cluster-to-cluster_{context}"]
+= Backing up data from one cluster and restoring it to another cluster
+
+In general, you back up data from one {product-title} cluster and restore it on another {product-title} cluster in the same way you back up and restore data to the same cluster. However, there are some additional prerequisites and differences in procedure when you back up data from one {product-title} cluster and restore it on another.
+
+.Prerequisites
+
+* All relevant prerequisites for backing up and restoring on your platform (AWS, Nicrosoft Azure, etc.) described in the relevant section of this guide, especially for the Data Protection Application (DPA).
+
+.Procedure
+
+. Make the following additions to the procedures given for your platform:
+
+** Ensure that the backup store location (BSL) and volume snapshot location have the same names and paths to restore resources to another cluster.
+** Share the same object storage location credentials across the clusters.
+** If you use the Velero `file-system-backup` option, enable the `--default-volumes-to-fs-backup` flag for use during backup by running the following command:
++
+[source,terminal]
+----
+$ velero backup create <backup_name> --default-volumes-to-fs-backup <any_other_options>
+----
+** For best results, use OADP to create the namespace on the destination cluster.
```

Comment:
@kaovilai @PrasadJoshi12 @shubham-pampattiwar 
Please check lines 22-27 of this file (oadp-backing-and-restoring-from-cluster-to-cluster.adoc).

Reply:
lgtm !